tests/wasmfs/wasmfs_getdents.out

```diff
@@ -50,17 +50,17 @@ d.d_off = 280
 d.d_reclen = 280
 d.d_type = directory
 
-d.d_name = stderr
```

Comment:
Was this order not deterministic before?

Reply:
It was deterministic, but didn't display in terms of insertion order.